.idea/codeStyles/Project.xml

```diff
@@ -47,6 +47,28 @@
       <option name="SPACE_BEFORE_SUPERCLASS_COLON" value="false" />
     </Objective-C>
     <Objective-C-extensions>
+      <option name="GENERATE_INSTANCE_VARIABLES_FOR_PROPERTIES" value="ASK" />	
```

Comment:
This file is not relevant to this PR so you can remove it from your version control

Reply:
We intentionally track that file. So, instead of removing the file itself, just revert the changes made to it.